### ISSUE TITLE
Invalid Playlist: The EXT-X-MEDIA-SEQUENCE tag MUST appear before the first Media Segment in the Playlist

### ISSUE THREAD

**Issue by mdanilakis** [kuu/hls-parser#36]
Hi @kuu,

I came across the following issue but I can't see what's wrong with parsing the playlist:

Error: Invalid Playlist : The EXT-X-MEDIA-SEQUENCE tag MUST appear before the first Media Segment in the Playlist.

```
#EXTM3U
# NeuLion Adaptive Streaming Server Version: 4.2.20181107.22941 
#EXT-X-PROGRAM-DATE-TIME:2018-12-05T23:41:56Z
#EXT-X-TARGETDURATION:10
#EXT-X-MEDIA-SEQUENCE:1116587
#EXTINF:10,
3y7ZyvdpWu_hd_800_20181205234156_042.ts
#EXTINF:10,
3y7ZyvdpWu_hd_800_20181205234206_042.ts
#EXTINF:10,
3y7ZyvdpWu_hd_800_20181205234216_042.ts
#EXTINF:10,
3y7ZyvdpWu_hd_800_20181205234226_042.ts
#EXTINF:10,
3y7ZyvdpWu_hd_800_20181205234236_042.ts
#EXTINF:10,
3y7ZyvdpWu_hd_800_20181205234246_042.ts
```

As far as i can tell the EXT-X-MEDIA-SEQUENCE appears before the first segment.

**Comment by kuu** [comment]
@mdanilakis Thanks for letting me know.
I think I fixed the bug at https://github.com/kuu/hls-parser/pull/37
And it's already published to npm as v0.2.3
Can you check if it solved your issue?
Thanks

**Comment by mdanilakis** [comment]
Thanks @kuu. This seems to have fixed the problem.
